application/views/admin/global_settings/_email.php

```diff
@@ -61,9 +61,9 @@
             'name' => 'emailsmtpssl',
             'value'=> getGlobalSetting('emailsmtpssl') ,
             'selectOptions'=>array(
-                ""=>gT("Off",'unescaped'),
-                "ssl"=>gT("SSL",'unescaped'),
-                "tls"=>gT("TLS",'unescaped')
+                ""=>gT("Off (insecure)",'unescaped'),
```

Comment:
"insecure" is wrong - should be 'unsafe' -same for other occurences.

Reply:
I understand that 'unsafe' could be more precise language wise, but term 'insecure' is now considered acceptable in IT security, see https://en.wikipedia.org/wiki/Transport_Layer_Security#Cipher

Also, this is inline how it looks in LS4. If you still think it is worth changing, I will make additional commit.

Reply:
I fixed it for LS4 already, please change it here, too.

Reply:
Done.